test/integration/admin_login_test.rb

```diff
@@ -14,8 +14,8 @@ def setup
                 role: 1)
     @category1 = Category.create(name: 'Hot Beverages')
     @category2 = Category.create(name: 'cold beverages')
-    @item1 = category1.items.create(title: 'espresso', description: "this is black gold", price: 30000)
-    @item2 = category2.items.create(title: 'cold pressed coffee', price: 8000, description: "hipster nonsense", price: 20000)
+    @project1 = category1.projects.create(title: 'espresso', description: "this is black gold", price: 30000)
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.